assets/extensions/examples/online-extensions.js

```diff
@@ -0,0 +1,105 @@
+/*
+  This file contains extensions being used by the OpenShift Online Developer Preview
+  They can be used as reference examples.
+*/
+
+/*
+  Override the CLI download locations
+*/
+window.OPENSHIFT_CONSTANTS.CLI = {
+  "Linux (32 bits)": "https://openshift.com/tbd_url/32bitlinux.tar.gz",
+  "Linux (64 bits)": "https://openshift.com/tbd_url/64bitlinux.tar.gz",
+  "Windows":         "https://openshift.com/tbd_url/win.zip",
+  "Mac OS X":        "https://openshift.com/tbd_url/mac.zip"
+};
+
+/*
+  Override the documentation links
+*/
+window.OPENSHIFT_CONSTANTS.HELP = {
+  "cli":                     "https://docs.openshift.com/online/latest/cli_reference/overview.html",
+  "get_started_cli":         "https://docs.openshift.com/online/latest/cli_reference/get_started_cli.html",
+  "basic_cli_operations":    "https://docs.openshift.com/online/latest/cli_reference/basic_cli_operations.html",
+  "webhooks":                "https://docs.openshift.com/online/latest/dev_guide/builds.html#webhook-triggers",
+  "new_app":                 "https://docs.openshift.com/online/latest/dev_guide/new_app.html",
+  "start-build":             "https://docs.openshift.com/online/latest/dev_guide/builds.html#starting-a-build",
+  "deployment-operations":   "https://docs.openshift.com/online/latest/cli_reference/basic_cli_operations.html#build-and-deployment-cli-operations",
+  "route-types":             "https://docs.openshift.com/online/latest/architecture/core_concepts/routes.html#route-types",
+  "persistent_volumes":      "https://docs.openshift.com/online/latest/dev_guide/persistent_volumes.html",
+  "compute_resources":       "https://docs.openshift.com/online/latest/dev_guide/compute_resources.html",
+  "default":                 "https://docs.openshift.com/online/latest/welcome/index.html"
+};
+
+/*
+  Custom angular module 
+*/
+angular.module('openshiftOnlineConsoleExtensions', ['openshiftConsole'])
+  .run(function(HawtioExtension) {
+
+    /*
+      Request system status from statuspage.io
+    */
+    var system_status_elem = $('<a href="http://status.openshift.com/" target="_blank" class="nav-item-iconic system-status project-action-btn" style="display: none;">');
+    var system_status_elem_mobile = $('<div row flex class="navbar-flex-btn system-status-mobile" style="display: none;">');
+
+    $.getJSON("https://m0sg3q4t415n.statuspage.io/api/v2/summary.json", function (data) {
+      var n = data.incidents.length;
```

Comment:
Yeah we decided to make a dedicated spot and recommended styles for someone
to plug in their own system status.

On Thu, Feb 11, 2016 at 3:28 PM, Sam Padgett <notifications@github.com>
wrote:

> In assets/extensions/examples/online-extensions.js
> <https://github.com/openshift/origin/pull/7169#discussion_r52661851>:
>
> > +};
> > +
> > +/*
> > +  Custom angular module
> > +*/
> > +angular.module('openshiftOnlineConsoleExtensions', ['openshiftConsole'])
> > +  .run(function(HawtioExtension) {
> > +
> > +    /*
> > +      Request system status from statuspage.io
> > +    */
> > +    var system_status_elem = $('<a href="http://status.openshift.com/" target="_blank" class="nav-item-iconic system-status project-action-btn" style="display: none;">');
> > +    var system_status_elem_mobile = $('<div row flex class="navbar-flex-btn system-status-mobile" style="display: none;">');
> > +
> > +    $.getJSON("https://m0sg3q4t415n.statuspage.io/api/v2/summary.json", function (data) {
> > +      var n = data.incidents.length;
>
> Maybe a null guard just in case: if (!data.incidents) return;?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7169/files#r52661851>.
>